bvm/ballerina-core/src/main/java/org/ballerinalang/model/types/BTableType.java

```diff
@@ -17,23 +17,34 @@
  */
 package org.ballerinalang.model.types;
 
-import org.ballerinalang.model.values.BDataTable;
+import org.ballerinalang.model.values.BTable;
 import org.ballerinalang.model.values.BValue;
 
 /**
  * {@code BDataTableType} represents a output data set of a SQL select query in Ballerina.
```

Comment:
Shall we change {@code BDataTableType} to {@code BTableType}  ?

Reply:
Fixed with 9f1d08c140b224ec932c0a0933457da9d3f401bf